docs/tutorials/services/data-mlmodel-tutorial.md

```diff
@@ -0,0 +1,322 @@
+---
+title: "Capture Data and Train a Model"
+linkTitle: "Capture Data and Train a Model"
+weight: 60
+type: "docs"
+description: "Configure data capture and cloud sync, filter and tag captured data, and train a new ML model."
+image: "/tutorials/data-management/data-captured-images.png"
+imageAlt: "The data page of the Viam app showing a gallery of the images captured from the Viam Rover."
+images: ["/tutorials/data-management/data-captured-images.png"]
+aliases:
+  - "/tutorials/data-management-tutorial/"
+  - "/tutorials/data-management/"
+  - "/manage/data-management/data-management-tutorial/"
+  - "/tutorials/services/data-management-tutorial/"
+tags: ["data management", "data", "mlmodel", "vision", "services", "try viam"]
+authors: []
+languages: []
+viamresources: ["data_manager", "mlmodel", "vision", "camera"]
+level: "Beginner"
+date: "8 February 2023"
+cost: "0"
+no_list: true
+# SMEs: Alexa Greenberg, Natalia Jacobowitz
+---
+
+In this tutorial, you will use three Viam services together to enable a smart machine to recognize specific objects in the world around it:
+
+- The [data management](#the-data-management-service) service, to capture images from a camera on your smart machine and sync them to the cloud.
+- The [ML model](#the-ml-model-service) service, to manage and deploy a machine learning (ML) model based on these images, once you added tags to the images matching the objects you want to detect.
+- The [vision](/services/vision/) service, to enable your smart machine's camera to detect objects defined in the ML model on its own.
+
+Working together, these services enable your smart machine to recognize its surroundings and make intelligent decisions about the world it sees.
+
+{{< alert title="Tip" color="tip" >}}
+To get started easily, you can rent a rover through [Try Viam](https://app.viam.com/try), which is pre-configured with everything you need to begin this tutorial.
```

Comment:
I'm not sure what easily is supposed to indicate. Maybe better to try this without any hardware?
```suggestion
To try this right now, you can rent a rover through [Try Viam](https://app.viam.com/try), which is pre-configured with everything you need to begin this tutorial.
```

Reply:
You're right -- even better to mention "without any hardware" directly, I think!